assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -201,25 +225,24 @@ function SizePositionPanel({ selectedElements, onSetProperties }) {
           onChange={handleNumberChange('rotationAngle')}
           disabled={isFill}
         />
-        {/** TODO: Implement flip horizontal mode */}
-        <FlipButton>
-          <FlipHorizontal />
-        </FlipButton>
-        {/** TODO: Implement flip vertical mode */}
-        <FlipButton>
-          <FlipVertical />
-        </FlipButton>
-        <Toggle
-          icon={<Fullbleed />}
-          value={state.isFill}
-          isMultiple={false}
-          onChange={(value) => {
-            setState({
-              ...state,
-              isFill: value,
-            });
-          }}
-        />
+        {canFlip && (
+          <>
+            <FlipControls onChange={handleFlipChange} value={state.flip} />
+            {isSingleElement && (
+              <Toggle
```

Comment:
Could non-flippable elements be fullbleed as well?

Reply:
Not currently, however, it's true that these are not really related and it could happen in the future, in theory. Will separate.